config/routes.rb

```diff
@@ -1,6 +1,15 @@
 Rails.application.routes.draw do
-  root ("articles#index") #get "/articles", to: "articles#index"
+  root ("home#index")
   resources :articles do
     resources :comments
   end
+  resources :users, except: %i[destroy] do
```

Comment:
@JsMontesR What does the resources helper method?

Reply:
The resources helper method actually enables all the CRUD bases endpoints for an specific model, it allows some aditional configuration for select only some kind of operations (for this example, all excepts the "destroy" capability).

Reply:
@JsMontesR Really good ✅.